tests/performance/apdex-score/README.md

```diff
@@ -214,14 +216,17 @@
 
 </details>
 
-- Find the android version by running `adb shell getprop | grep ro.build.version.release`
-- Find the device name by running `adb shell getprop | grep ro.product.model`
-
-4. Set the environment variable `APDEX_TEST_SETTINGS` with the path of your settings file.
-
-```sh {"id":"01J2WE5FDN0D40ZJ5XA7ZHHH4Z"}
-export APDEX_TEST_SETTINGS=/Users/pepe/Documents/apdex-settings.json
-```
+4. Set the environment variable `APDEX_TEST_SETTINGS` with the path of your settings file (apdex-settings.json).
+    ```
+    export APDEX_TEST_SETTINGS=/Users/pepe/Documents/apdex-settings.json
+    ```
+   - Ensure the `apdex-settings.json` file has been updated with the correct instance url, login credentials and assertion texts for page navigation, forms and other app interactions.
+   - Under the skip section of the settings file, set `true` for the tests you want to skip and `false` for those you want to execute.
+   - Update the fields for `platformVersion` and `deviceName` to match the value for your device.
```

Comment:
What about the `appPath`? If the app is already installed on the phone can we use that app or do we need a phone that doesn't have an app installed to install it using that apk? That information should be in the prerequisites section and here you should specify where you can find an apk - you can use [the link to the cht-android releases](https://github.com/medic/cht-android/releases) for example